### ISSUE TITLE
efficientnet-d0 pretrained is very bad 

### ISSUE THREAD

**Issue by chuanfuye** [zylo117/Yet-Another-EfficientDet-Pytorch#432]
Get the pre-trained model from effieientnet-b0, (https://github.com/lukemelas/EfficientNet-PyTorch/releases/download/1.0/efficientnet-b0-355c32eb.pth)) 
then set the parameters in read.me according to you, and there is no change in learning rate and epoch.
I get efficientdet-d0_9_147850.pth, but the coco test result is very bad, 
my device is also a single card 2080ti, the test is also based on your settings in the readme, and then set the epoch to 30, but the effect is still very bad, can you share any training skills? For example, data processing.Learning rate etc.Thank you very much. Below is my execution code. look forward to your reply.

python train.py -c 2   --batch_size 8 --lr 1e-5 --num_epochs 10 \
     --load_weights / efficientnet-b0.pth
     --head_only True
 python coco_eval.py -c 0 -w efficientdet-d2_9_147850.pth
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.003
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.008 
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.002
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.001
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.002
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.005
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.016
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.029
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.032
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.002
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.016
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.052
 Is my pre-training model downloaded wrong?

**Comment by zylo117** [comment]
`-c 2` means training a D2 model but you loaded a D0 weights.

**Comment by chuanfuye** [comment]
Thank you for such a quick reply. Sorry, the question I submitted was wrong. This is it. 
python train.py  --batch_size 8 --lr 1e-5      --load_weights weights/efficientnet-b0.pth
The default is -c 0
How long did you train at that time please, If the D0 single card is 2080ti,.my training loss is still relatively high 0.848. I need to continuously adjust the learning rate dynamically.or Data preprocessing，Thanks

**Comment by zylo117** [comment]
0.8 is not even close to the final loss. You can refer to FAQ Q2. It's underfitting, try a much greater lr.

https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/issues/386
